kludges.js

```diff
@@ -422,18 +424,10 @@ function init() {
         registerOverrideProp(imports.ui.viewSelector, "PINCH_GESTURE_THRESHOLD", 0)
     }
 
-    const overviewSwipeTracker = Main.overview._swipeTracker
-    if (overviewSwipeTracker) {
-        registerOverrideProp(overviewSwipeTracker, "enabled", false)
-    }
-
-    const workspaceSwipeTracker = Main.wm._workspaceAnimation?._swipeTracker
-    if (workspaceSwipeTracker) {
-        registerOverrideProp(workspaceSwipeTracker, "enabled", false)
-    }
-
-    if (Main.wm._swipeTracker)
-        registerOverrideProp(Main.wm._swipeTracker._touchpadGesture, "enabled", false);
```

Comment:
Is it correct to not include this in the swipeTrackers list? I would rather include to much than to little so we maybe support more gnome versions.

Reply:
Ah, good pickup!  Yes, in (at least Gnome 43) it looks like it was moved to `Main.wm._workspaceAnimation?._swipeTracker` but yes, will include it there too.  The filter on `swipeTrackers` will filter it out if it's undefined (which it will be since we'll use an optional chain on it).

Thanks @Lythenas!

Reply:
Updated (re-added back in).